### ISSUE TITLE
TypeScript error when using pretrained JSON data

### ISSUE THREAD

**Issue by DenisTimofijuk** [BrainJS/brain.js#696]
## _What_ is wrong?

I exported training results in to JSON file, but when I try to load them, typescript is complaining, although product is working fine.

## _How_ do we replicate the issue?

Step 1. train your network
Step 2. save your results to file `const json = net.toJSON();`
Step 3 load your results from file `net.fromJSON(json);`

## Expected behavior (i.e. solution)

I think TypeScritp should not complain.

## Other Comments

```
Argument of type '{ sizes: number[]; layers: ({ "0": { bias?: undefined; weights?: undefined; }; "1": { bias?: undefined; weights?: undefined; }; "2": { bias?: undefined; weights?: undefined; }; "3": { bias?: undefined; weights?: undefined; }; ... 35 more ...; "39"?: undefined; } | ... 5 more ... | { ...; })[]; outputLookup: boolean;...' is not assignable to parameter of type 'INeuralNetworkJSON'.
  Types of property 'activation' are incompatible.
    Type 'string' is not assignable to type 'NeuralNetworkActivation'.ts(2345)
```

[Saving training to file](https://github.com/DenisTimofijuk/snowpack-brain.js/blob/main/src/helper/saveTrainedData.ts)
[Pretrained data file](https://github.com/DenisTimofijuk/snowpack-brain.js/blob/main/src/JSON/pretrained/data.v0.json)  
[loading pretrained data from file](https://github.com/DenisTimofijuk/snowpack-brain.js/blob/main/src/modules/brain-handler.ts)

**Comment by mubaidr** [comment]
Typescript conversion is still in progress, we are looking for contributions: https://github.com/BrainJS/brain.js/issues/684

**Comment by robertleeplummerjr** [comment]
This should now be resolved with at least version 2.0.0-beta.12.  <img width="376" alt="Screen Shot 2022-04-13 at 10 17 12 AM" src="https://user-images.githubusercontent.com/679099/163201060-70fc1a42-3901-4265-9581-7587359635f3.png">
